src/entity/base_entity.rs

```diff
@@ -299,15 +299,56 @@ pub trait EntityTrait: EntityName {
     /// let insert_result = cake::Entity::insert(apple).exec(&db).await?;
     ///
     /// assert_eq!(insert_result.last_insert_id, 15);
-    /// // assert_eq!(insert_result.rows_affected, 1);
     /// #
     /// # Ok(())
     /// # });
     ///
     /// assert_eq!(
     ///     db.into_transaction_log(),
     ///     vec![Transaction::from_sql_and_values(
-    ///         DbBackend::Postgres, r#"INSERT INTO "cake" ("name") VALUES ($1) RETURNING "id""#, vec!["Apple Pie".into()]
+    ///         DbBackend::Postgres,
+    ///         r#"INSERT INTO "cake" ("name") VALUES ($1) RETURNING "id""#,
```

Comment:
I don't get it. Shouldn't it return all columns here?

Reply:
Actually, this is unchanged. I just added a new line after `DbBackend::Postgres,`. For Postgres, it use `RETURNING` to get the last inserted id.

Reply:
While other db get the last insert id out of `execute`.

Reply:
Oh so we have 2 different insert implement here?

Reply:
> Oh so we have 2 different insert implement here?

Correct! And this happens on 0.3.x as well

Reply:
Oh then can I also locate the other test case?

Reply:
Please find it below. I have added a new review